### PR TITLE
Initial implementation of support for NAMD binary files. This is not hea...

### PR DESCRIPTION
**Pull request by radakb**
...vily

tested and is mostly designed for a few specific tasks, such as adding or
removing blocks of atoms from .coor and .vel files.

### CODE REVIEW

File: chemistry/namd/namdbinfiles.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        return self.values
+
+
+class NamdBinVel(NamdBinFile):
+    """Class to read or write NAMD "binvelocities" files.
+
+    NAMD internal units are assumed. These can be converted to 
+    Angstrom / picosecond by multiplying by NamdBinVel.PDBVELFACTOR.
+    """
+    PDBVELFACTOR = 20.45482706
```
Comment (swails) [inline chemistry/namd/namdbinfiles.py]: I don't see where this is used.

File: chemistry/namd/namdbinfiles.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    @property
+    def velocities(self):
+        return self.values
```
Comment (swails) [inline chemistry/namd/namdbinfiles.py]: Shouldn't this be

```Python
@property
def velocities(self):
    return self.values * self.PDBVELFACTOR
```

?

Comment (radakb) [inline chemistry/namd/namdbinfiles.py]: I suppose that might be expected behavior. I figured it could be left to the user.

**Comment by swails** [comment]
This seems to be superseded by #108.  For future reference, you can push more commits to a branch you issued a PR from to add to that pull request (i.e., you don't need to create a new branch and issue a new PR every time you want to make a change).

Closing in favor of #108.

**Pull request closed without merging**